versions/development/SPEC.md

```diff
@@ -597,7 +587,6 @@ runtime {
 |            | Inequality            | left-to-right | x!=y                 |
 | 3          | Logical AND           | left-to-right | x&&y                 |
 | 2          | Logical OR            | left-to-right | x\|\|y               |
-| 1          | Assignment            | right-to-left | x=y                  |
```

Comment:
I don't see the purpose of removing this. This might be obvious, but it is still part of the precedence order. If this does end up being removed, the precedence column should be adjusted for the other rows as well.

Reply:
In WDL, '=' is not an operator. It is part of the syntax of declaration. `name = expression` Most languages do not have '=' as an operator, C and C++ (Java?) being the well known exceptions.

Reply:
Okay, the precedence column will still need adjustment in that case :wink:.